BaseFilter/sections/foreign.txt

```diff
@@ -3092,6 +3092,9 @@ bb.is$$div[class="banner"]
 !---------------------------------------------------------------------
 ! NOTE: Indian
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/176683
+irctc.co.in###cube
+irctc.co.in##[id^="adContainer"]
```

Comment:
are not elements `div`'s?

Reply:
@Alex-302 they are `div`s

Reply:
should i add `div` tag selector before attr selector?

Reply:
I guess so. `div[id^="adContainer"]`